### PR TITLE
Create-Survey-Form

### PR DESCRIPTION
**Pull request by AnsarIbrahim**
### This PR adds the following sections :
- header
- tagline
- form

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @AnsarIbrahim, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project without reviewing yourself. kindly perform a code review to complete this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AnsarIbrahim** [comment]
I don't understand can you guide me pls

**Comment by GedeonTS** [comment]
Hi @AnsarIbrahim , In order to perform a code review, kindly click on the `File changed` above and write at least two comments, don't forget to use an inline comment referring to Microverse material.

Screenshot from 2023-02-20 15-00-26

**Review by AnsarIbrahim — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all required changes

Submit for re-review when you are done

Thanks 😁

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @AnsarIbrahim  👋 , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there! 

so_close

## Highlight
- Good use of GitHub features ✔️ 

## Required Changes ♻️
- According to the project requirement in consistency with a lesson you have gone through on [How to Give a Code Review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md), especially in the Use of GitHub Features, you are missing something at the moment.
- [ ] Kindly watch [this video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) and see how to make use of inline comments to further make your suggestions more clear to the reviewed colleague. Highlighting some portions and giving opinions on them makes it easier for him to know where the code needs improvement.
- [ ] Kindly make your suggestions consistent with the identified problem. All should be in a descriptive sentence where you explain `what` was not done to standard, suggest or guide students on `how` to fix it, and `why` the change was necessary. This allows you to mentor him correctly rather than just standing a chief judge to weigh his imbalances 😄 
- [ ] An important principle is to communicate politely with the student or colleague. Try not to use _Command tones/prompts_ nor _order him around_. The use of kind words like: "It appears your linters are not passing now due to some syntax error. Kindly check to be sure...." etc.
- It's a requirement that you add at least 2 inline comments. Just a kind reminder, do not forget to keep your communication really friendly and politely with clarity just as you did here 😉 

Let's do it again, Champ! 💪 

## Optional suggestions
- You can consider using memes, gifs, and smileys like 😉 😄 in order to make it more fun and engaging. A little of laughter will make the correction really sink. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey-From</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="title">freeCodeCamp Survey Form</h1>
+    <p class="description"><i>Thank you for taking the time to help us improve the platform</i></p>
```
Comment (AnsarIbrahim) [inline index.html:12]: Please create a header tag and put it inside this h1 and p tag, so look like professional and clean code 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        </div>
+
+        <div class="info-div">
+            <label for="">Any comments or suggestions?</label><br>
+            <textarea class="input-class" name="" id="" cols="30" rows="10" placeholder="Enter your comment here..."></textarea><br>
+        </div>
+
+        <input class="submit" type="submit" name="" id="submit" value="Go Submit"><br>
+    </form> 
+    </div>
```
Comment (AnsarIbrahim) [inline index.html:124]: Please use main and section tags inside because it looks like a clean and professional code 😄

**Review by AnsarIbrahim — COMMENTED** [review]
# Requried Changes ♻️ 

Please check my comments below and make all required changes 👍 

Submit to re-review when youre done 

Thanks  😁

**Review by binhussen — APPROVED** [review]
Hi @AnsarIbrahim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions
- [Optional] Great Job for fulfilling previous suggestions 👍 I think it would better to give comments as match as possible becouse **This project's goal is to learn how to use GitHub features for reviews and how to communicate professionally.**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**